packages/server/postgres/queries/removeIntegrationSearchQueryToPG.ts

```diff
@@ -0,0 +1,7 @@
+import getPg from '../getPg'
+import {removeIntegrationSearchQueryQuery} from './generated/removeIntegrationSearchQueryQuery'
+
+const removeIntegrationSearchQueryToPG = async (id: number, userId: string, teamId: string) => {
```

Comment:
-1 That name is inconsistent with otheres, can we just name it `removeIntegrationSearchQuery` please?